Source/DafnyLanguageServer.Test/Various/ConcurrentInteractionsTest.cs

```diff
@@ -73,6 +84,70 @@ public class ConcurrentInteractionsTest : DafnyLanguageServerTestBase {
       Assert.AreEqual(1, diagnostics.Length);
     }
 
+    [TestMethod, Timeout(MaxTestExecutionTimeMs)]
+    public async Task VerificationErrorDetectedAfterCanceledSave() {
+      // Create a document that'll be slightly slow to verify
```

Comment:
Would it help if we add an option to Boogie that'll make the verifier take forever? Similar to this https://github.com/boogie-org/boogie/pull/472

Reply:
That could be useful, though I don't think it's urgent.